features/step_definitions/journey_test_steps.rb

```diff
@@ -1488,22 +1501,73 @@ def filter_to_check(filter_name,filter_value,filter_exist)
     page.should have_link('Log out')
 end
 
+Then /I am presented with the Service updates page$/ do
+  todays_date = Time.new.strftime("%A %d %B %Y")
+  page.find(:xpath,"//p[contains(text(), 'Activity for')]/../h1[contains(text(), '#{todays_date}')]")
+  page.should have_selector(:xpath, "//*/div/label[@for='audit_date'][contains(text(), 'Audit Date')]")
+  page.should have_selector(:xpath, "//*[contains(@id, 'audit_date') and contains(@placeholder, 'eg, 2015-07-23')]")
+  page.should have_selector(:xpath, "//*/div[@class='option-select-label'][contains(text(), 'Show')]")
+  page.find(:xpath, "//*/div[@class='options-container']//label[@for='acknowledged-1']/input[@type='radio']/..").text().should match('All')
+  page.find(:xpath, "//*/div[@class='options-container']//label[@for='acknowledged-2']/input[@type='radio']/..").text().should match('Acknowledge')
```

Comment:
I think these need a `d` on the end:
![screen shot 2016-01-08 at 18 04 32](https://cloud.githubusercontent.com/assets/6525554/12205409/57f29442-b632-11e5-9ea4-a8754a762013.png)